packages/app/tests/Api/Unit-Test.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * Test file for the Unit react element
+ *
+ * @package: REA app
+ * @author:  Steven Fontaine <fontainesw@msoe.edu>
+ * @since:   2019-01-18
+ */
+
+import React from 'react';
+import { createRenderer } from 'react-dom/test-utils';
+import createComponent from 'react-unit';
+import tape from 'tape';
+import addAssertions from 'extend-tape';
+import jsxEquals from 'tape-jsx-equals';
+const test = addAssertions(tape, {jsxEquals});
+
+// Component to test
+import {Unit, UnitList, GetSingleUnit}  from '../../pages/Api/unit';
+
+
+test('UnitDisplaysUnit', (t) => {
```

Comment:
t might not be the best variable for readability. It is not clear what it stands for, without knowing what the "test" function takes as parameters

Reply:
That is the generic for the function as described in the tape docs